### DISCUSSION TITLE
Parasite (Neutral) & Symbiote (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#791] [Role Ideas]
**Description:** The Parasite and Symbiote are both third party roles who can enter the bodies of players, and can force them to make inputs against their will.

These roles deal no damage, but start with a knife that allows you to jump into any player once you melee them. Once inside of them, you can make any input for them and control what they do if you choose to do so. Your goal is to now use them to your advantage.
Be careful with your host though, you will die if they do.
After a short time, you also slowly lose health when not inside of a host.

_The Symbiote_ alerts the host once they enter, and can now only talk to their host. The host can't directly talk to the Symbiote without talking out loud to themselves though. The Symbiote gains access to a quick recharging radar to help direct and alert their host of possible threats.

_The Parasite_, on the other hand, does not alert the host when you enter them, and you cannot talk to your host. The Parasite has the radar the Symbiote has, but also has access to the ability to swap to another body within a short radius and on a cooldown. The Parasite cannot sit in a single host forever though, since after a while his host will move slower, turn purple, and a while after that the host will start taking damage over time.

**Win Condition:** The Symbiote wins with whoever they are inside, while the Parasite wins by being inside the last surviving player.

The Symbiote is meant to be a fun buddy co-op experience with a buddy of your choice, while the Parasite is meant to be a chaotic pest to keep your eye on, since he can steal a win from under your nose.

Idea by: @SmungDaal

**Comment by YG188** [comment]
wow this is a really interesting idea you have there!
